packages/client-app/src/libs/sendRequest.ts

```diff
@@ -7,6 +7,11 @@ import type {
 import type { AxiosError, AxiosRequestConfig } from 'axios'
 import axios from 'axios'
 
+/**
+ * Convert the parameters to an object
+ * @param RequestInstanceParameter
+ * @returns Record<string, string>
```

Comment:
we actually don't need these when using typescript! Only useful if you want to actually describe the parameter, just the type can be inferred from typescript

Reply:
removed we are so back